deps/src/main/java/com/microsoft/azure/sdk/iot/deps/transport/amqp/AmqpsOperations.java

```diff
@@ -1,13 +0,0 @@
-package com.microsoft.azure.sdk.iot.deps.transport.amqp;
-
-import org.apache.qpid.proton.engine.Link;
-import org.apache.qpid.proton.engine.Session;
-
-import java.io.IOException;
-
-public abstract class AmqpsOperations
```

Comment:
I don't see this being used. @timtay-microsoft Do you think it's safe to remove?

Reply:
Yes, looks safe to remove.

Reply:
Yep, go ahead and remove it